package.json

```diff
@@ -19,11 +19,15 @@
     },
     "devDependencies": {
         "browserify": "^11.2.0",
-        "watchify": "^3.4.0"
+        "watchify": "^3.4.0",
+        "hbsfy": "^2.4.1",
+        "gulp": "^3.9.0",
+        "gulp-sass": "^2.1.1"
```

Comment:
Awesome job looking up documentation and getting gulp to work! Googling is what I do 90% of the time.